packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { initializeIcons } from "@fluentui/react";
+
+let iconsInitialized = false;
+
+/**
+ * Initialize Fluent icons used this library's components, if they have not already been initialized.
+ */
+export function initializeFluentUiIcons(): void {
+	if (!iconsInitialized) {
+		initializeIcons();
+		iconsInitialized = true;
```

Comment:
I think we can return the boolean value here to check whether icon has been initialized.

Reply:
By the time this function returns, that will always be true. Is there a particular use-case you're thinking of?

Reply:
we just want to make sure the icon only initialized once all the time, right? Is that other place call initializeFluentUiIcons()?

Reply:
Right. So yeah, any module containing components that use FluentUI icons should call this.